stripe-ruby-mock.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'stripe', '~> 1.31'
+  gem.add_dependency 'stripe', '~> 2.8'
```

Comment:
``` 
=== 2.0.3 2017-03-16
* Fix marshalling of `StripeObjects` that have an embedded client
```
I've tested it with 2.0.3 and tests didn't fail, but failed with 2.0.2. So I think that's the issue that causes some specs to fail. I've lowered required version to 2.0.3